src/Admin/Enums/HtmlHelperExtensions.cs

```diff
@@ -0,0 +1,19 @@
+﻿
+using Bit.SharedWeb.Utilities;
+
+// ReSharper disable once CheckNamespace
+namespace Microsoft.AspNetCore.Mvc.Rendering;
+
+public static class HtmlHelper
+{
+    public static IEnumerable<SelectListItem> GetEnumSelectList<T>(this IHtmlHelper htmlHelper, IEnumerable<T> values)
```

Comment:
The issue I had with it was that I needed to pass the type. But then it would return everything. Now I just pass the enum values we want converted for the drop down.